### PR TITLE
docs - update sso

### PR DESCRIPTION
**Pull request by nllho**
Bit of clean up across SSO & SAML docs -- rather pleased that deletions > additions in this PR :)

Changes:
- Moved stuff from Docs to Troubleshooting
  - Moved "Checking if SSO is working correctly" from SSO doc to "Can't log in" page.
  - Moved "Troubleshooting tips" from SAML doc to "Troubleshooting SAML" page.
- Simplified the "Can't log in" page (I'd added an excess of headers last time).

### CODE REVIEW

File: docs/administration-guide/10-single-sign-on.md [hunk 1]
```diff
@@ -134,17 +134,14 @@ Once a person creates an account, you cannot change the authentication method fo
 
 Note that you must have at least one account with email and password login. This account safeguards you from getting locked out of your Metabase if there are any problems with your SSO provider.
 
-## Checking if SSO is working correctly
+## Troubleshooting
```
Comment (jeff-bruemmer) [inline docs/administration-guide/10-single-sign-on.md:137]:
```suggestion
## Troubleshooting login issues
```

File: docs/enterprise-guide/authenticating-with-saml.md [hunk 2]
```diff
@@ -225,6 +213,10 @@ When users log in to Metabase for the first time via SSO, this will automaticall
 
 You can find example code that uses SAML authentication in the [SSO examples repository](https://github.com/metabase/sso-examples).
 
+## Troubleshooting
```
Comment (jeff-bruemmer) [inline docs/enterprise-guide/authenticating-with-saml.md:216]:
```suggestion
## Troubleshooting SAML issues
```

File: docs/enterprise-guide/saml-google.md [hunk 3]
```diff
@@ -23,4 +22,8 @@ For more information, check out our guide for [authenticating with SAML](authent
 | SAML Identity Provider URL          | Google Admin console > Google Identity Provider details > Copy the **SSO URL**                                                                                |
 | SAML Identity Provider Certificate  | Google Admin console > Google Identity Provider details > Download certificate                                                                                |
 | SAML Application Name               | Google Admin console > Google Identity Provider details > Copy the **Entity ID**                                                                              |
-| SAML Identity Provider Issuer       | Google Admin console > Google Identity Provider details > Download metadata                                                                                   |
\ No newline at end of file
+| SAML Identity Provider Issuer       | Google Admin console > Google Identity Provider details > Download metadata                                                                                   |
+
+## Troubleshooting
```
Comment (jeff-bruemmer) [inline docs/enterprise-guide/saml-google.md:27]:
```suggestion
## Troubleshooting SAML issues
```

File: docs/enterprise-guide/saml-keycloak.md [hunk 4]
```diff
@@ -65,4 +64,8 @@ Let's say we want email, name, and surname to be passed between the client (Meta
     - **SAML Attribute Name**: the name that Metabase expects to receive.
     - **SAML Attribute NameFormat**: select “Basic” from the dropdown menu.
 
-You can find the attribute values from your Metabase **Admin settings** > **Authentication** > **SAML** > **Attributes**.
\ No newline at end of file
+You can find the attribute values from your Metabase **Admin settings** > **Authentication** > **SAML** > **Attributes**.
+
+## Troubleshooting
```
Comment (jeff-bruemmer) [inline docs/enterprise-guide/saml-keycloak.md:69]:
```suggestion
## Troubleshooting SAML issues
```

File: docs/troubleshooting-guide/saml.md [hunk 5]
```diff
@@ -8,6 +8,10 @@ title: Troubleshooting SAML authentication setup
 
 Some common problems when setting up SAML.
 
+## Does your app support SAML?
+
+Verify that the application you created in your IdP supports SAML. Sometimes other options are presented during the app creation process.
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/saml.md:13]: Like what other options? Can we give examples?

File: docs/troubleshooting-guide/saml.md [hunk 6]
```diff
@@ -31,6 +35,10 @@ After filling out the authentication form with your identity provider, you go ba
 1. You should have received an XML file from your identity provider. Open that metadata file, and check to make sure the certificate you inputted is correct.
 2. Go to **Admin settings** > **Settings** > **Authentication** > **SAML**. Check that the certificate that you entered into the **SAML Identity Provider Certificate** field matches the certificate in the XML file you got from your identity provider. Depending on your provider, you might need to download this, open it in a text editor, then copy and paste the certificate's contents into the field.
 
+## Is the SSO URL correct?
+
+Verify that the Single Sign On URL (or equivalent) that you enter on your SAML provider’s website has `/auth/sso` appended to it. For instance, if you want your users to end up at `https://metabase.mycompany.com`, the full URL should be `https://metabase.mycompany.com/auth/sso`.
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/saml.md:40]: Good example here.

**Review by jeff-bruemmer — APPROVED** [review]
Big thumbs up on the cleanup here. Nice work, @nllho.

File: docs/troubleshooting-guide/saml.md (hunk 6 above)
Comment (nllho) [reply docs/troubleshooting-guide/saml.md:40]: I moved this paragraph from the original SAML doc, so credit for this example probably goes back to you ;)

File: docs/troubleshooting-guide/saml.md (hunk 5 above)
Comment (nllho) [reply docs/troubleshooting-guide/saml.md:13]: I moved this paragraph from "Troubleshooting tips" (originally in `authenticating-with-saml.md`), so  I don't have this info on hand.

[Added a note to the backlog](https://www.notion.so/metabase/Updates-by-topic-parent-ticket-5f93c492e1b8483ab5500768b10af0f9#9d4cf3ede11a40cdae702eeeb50a6592) for now :)

**Pull request merged**